1-js/99-js-misc/06-unicode/article.md

```diff
@@ -1,172 +1,167 @@
 
-# Unicode, String internals
+# Unicode, detalhes internos de Strings
```

Comment:
It should be "das Strings" because "String" is singular and "Strings" plural.

Reply:
Strings, both singular and plural, can be referenced with the preposition "de".

There are some examples of this in both singular and plural in the pt-br version of [MDN docs - Strings](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/String)

Reply:
Well, we are not translating to Brazilian Portuguese, only Portuguese.